packages/babel-plugin-transform-es2015-parameters/README.md

```diff
@@ -72,3 +72,21 @@ require("babel-core").transform("code", {
   plugins: ["transform-es2015-parameters"]
 });
 ```
+
+## Options
+
+### `loose`
+
+`boolean`, defaults to `false`.
+
+In loose mode, parameters with default values will be counted into the arity of the function. This is not spec behavior where these parameters do not add to function arity:
```

Comment:
An explanation of why you'd want this might be a good addition here:

This is a more performant solution as JavaScript engines will fully optimize a function that doesn't reference `arguments`. Please do your own benchmarking and determine if this option is the right fit for your application.